### ISSUE TITLE
Support for mssql-django

### ISSUE THREAD

**Issue by valentinschabschneider** [joke2k/django-environ#295]
There is an official django-mssql-backend port by Microsoft in the making.
To use it you have to provide "mssql" as the engine.
https://github.com/microsoft/mssql-django#installation

Currently django-environ maps "mssql" to "sql_server.pyodbc".
https://github.com/joke2k/django-environ/pull/157

Don't know what the solution is, but it would be nice if there's a way to configure mssql-django in the next release.

**Comment by valentinschabschneider** [comment]
Either remove the mapping from Env.DB_SCHEMES or add another one like "mssql2".

Hotfix:
```python
env.DB_SCHEMES.pop("mssql")
```

**Comment by MartinSchmidt123** [comment]
I can confirm that this is still an issue.
But the workaround works great.

**Comment by hairypalm** [comment]
Yes, still an issue.  dj-database-url maps **mssql** to **mssqlms**:  https://pypi.org/project/dj-database-url/

This workaround works, but would be nice to have a documented way to connect using mssql engine.  Even documenting this workaround would be good.

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — there's a one-line workaround already shared in this thread, so it might be a candidate for closure or a docs-only follow-up.

Evidence:
- Your own [comment](https://github.com/joke2k/django-environ/issues/295#issuecomment-845891455) provides the workaround: `env.DB_SCHEMES['mssql'] = 'mssql'` (or `.pop('mssql')` then set), since `DB_SCHEMES` is a mutable class attribute at [`environ/environ.py`:128-151](https://github.com/joke2k/django-environ/blob/develop/environ/environ.py#L128-L151).
- @MartinSchmidt123 and @hairypalm [confirmed](https://github.com/joke2k/django-environ/issues/295#issuecomment-1499747109) the workaround works.
- Changing the default mapping is a breaking change (existing `sql_server.pyodbc` users would silently switch backends); the per-project override is the safer pattern.

If you'd like the workaround surfaced in docs instead, that's also worth a separate doc-only issue. Otherwise would you mind closing this out?

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the references above were verified manually.*
